webpack.config.js

```diff
@@ -33,6 +33,9 @@ const USE_HOT_MODULE_RELOAD =
 const NO_DEV_SERVER = env.NO_DEV_SERVER;
 const FORCE_WEBPACK_DEV_SERVER = env.FORCE_WEBPACK_DEV_SERVER;
 const IS_CI = !!env.CI || !!env.TRAVIS;
+const NO_TS_FORK = env.NO_TS_FORK === 'true';
+
+const IS_DEV_ENV = !IS_PRODUCTION || !IS_CI || !NO_TS_FORK;
```

Comment:
I'd say `IS_DEV_ENV` being affected by `NO_TS_FORK` is confusing and a mix of concerns. I think the following would be easier to reason about:

```js
const DEV_MODE = !(IS_PRODUCTION || IS_CI);
const SHOULD_FORK_TS = DEV_MODE && !env.NO_TS_FORK;
```